### DISCUSSION TITLE
Default Execution Endpoint Port/Storage

### DISCUSSION THREAD

**Discussion by Autoparallel** [paradigmxyz/reth#949] [Help]
I'm trying to checkpoint sync with lighthouse as my consensus client but can't reach reth as an eth1 backend via engine api. Is the default port set to 8551? Also, where is the default storage location for jwt hex?

I looked through the codebase for a while, but I could not find these so I figured I'd ask here. Pointing it out in the codebase could be helpful and I'm happy to contribute to the docs or CLI parsing for this.

**Comment by benharbit** [comment]
the default port is 8551

Default jwt.hex file location is here. https://github.com/paradigmxyz/reth/blob/238eea37cf23bd212823d76d2d07065a59ce7e2a/bin/reth/src/node/mod.rs#L365
